vision/lenet_mnist/lenet_mnist.jl

```diff
@@ -23,12 +22,11 @@ function LeNet5(; imgsize=(28,28,1), nclasses=10)
     out_conv_size = (imgsize[1]÷4 - 3, imgsize[2]÷4 - 3, 16)
```

Comment:
Not sure if the intent is just bug fixes or a refresh. If it is the former, then ignore this comment. If it is the latter, then we should use `Flux.outputsize` here instead of manually calculating the output size.

Reply:
This pull request was just to get the model running on the next julia release. I will add your comment to the list in the tracking issue though.